src/index.html

```diff
@@ -1,13 +1,54 @@
 <!doctype html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+<html lang="en" class="page">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div class="section section--block" data-qa="card">
+      <div class="product">
+        <a href="#">
+          <img src="./images/imac.jpeg"
+            alt="product"
+            class="product__image"
+          >
+        </a>
+        <h1 class="product__item">
+        Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+        </h1>
```

Comment:
Contents should have 2-space indentation comparing to parent tag
```suggestion
        <h1 class="product__item">
          Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
        </h1>
```